de/tests/test_factorization.py

```diff
@@ -4,7 +4,7 @@
 import pytest
 import numpy as np
 from scipy.special import expit
-from ..factorization import factorizeEstimate, alpha
+from ..factorization import MatrixSubtraction, cellLineComparison, factorizeEstimate, alpha, cellLineFactorization
```

Comment:
The functions `cellLineComparison` and `cellLineFactorization`, and `MatrixSubtraction` are in the fitting file. They should be imported from that.